doc/sphinx/installation.rst

```diff
@@ -37,7 +37,7 @@ The following tools libraries, including header files, are required to be able
 to compile and use |es|:
 
 CMake
-    The build system is based on CMake
+    The build system is based on CMake. At least version 3.11 is required.
```

Comment:
Do you really want to document version requirements twice? In CMakeLists.txt and in the documentation?

Reply:
i think it would be a better idea to give a short hint for various reasons if the configuration of espresso fails. Such information is prone to get outdated fast.

Reply:
Latest CMake versions (3.11+) usually do a good job of telling which dependencies are not met. We could also have a FAQ on the wiki for common issues (see #3947). I've removed the version numbers in 958d35d.